cmake/custom/static_library.cmake

```diff
@@ -0,0 +1,26 @@
+#.rst:
+#
+# Enables creation of static/shared library.
+#
+# autocmake.yml configuration::
+#
+#   docopt: "--static Create only the static library [default: False]."
+#   define: "'-DSTATIC_LIBRARY_ONLY={0}'.format(arguments['--static'])"
+
+
+option_with_print(
```

Comment:
Does the `option_with_print` work as expected now?

Reply:
Settings go through correctly, but the value printed to screen is always the one set as default. I am working on a fix Autocmake-side.